src/command/ping-command.ts

```diff
@@ -160,13 +169,13 @@ export class PingCommand implements CommandInterface<PingOptions> {
 			resolvedHostname: input.resolvedHostname ? input.resolvedHostname : null,
 			timings: input.timings ?? [],
 			stats: {
-				min: input.stats?.min ?? null,
-				max: input.stats?.max ?? null,
-				avg: input.stats?.avg ?? null,
-				total: input.stats?.total ?? null,
-				loss: input.stats?.loss ?? null,
-				rcv: input.stats?.rcv ?? null,
-				drop: input.stats?.drop ?? null,
+				min: (input.stats?.min || input.stats?.min === 0) ? input.stats?.min : null,
```

Comment:
Doesn't this do exactly the same?

Reply:
`input.stats?.min` may be also equal to `NaN`, which is accepted by `??`.